service/src/main/java/bio/terra/workspace/app/configuration/spring/BeanConfig.java

```diff
@@ -22,13 +24,49 @@ public NamedParameterJdbcTemplate getNamedParameterJdbcTemplate(
     return new NamedParameterJdbcTemplate(config.getDataSource());
   }
 
+  public static class HTMLCharacterEscapes extends CharacterEscapes {
```

Comment:
Presumably all of our services will want to do this. Did you consider putting it in the common lib?

Reply:
Part of the problem is that this is one setting for the Jackson `ObjectMapper`, and right now each service creates its own `ObjectMapper` object. I think we could reasonably provide a pre-configured one as a component in TCL including this change, but that's a bit out of scope here (filed PF-1424 to track it, though)